addons/rose/app/styles/rose/_typography.scss

```diff
@@ -12,60 +12,26 @@ html {
 }
 
 body {
+  // Used by Rose::Tabs and '/' in breadcrumbs
```

Comment:
thanks for adding these comments, it's helpful to understand the reasoning behind keeping them in here